js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -676,6 +676,9 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
 
         this.setAria('selected', nodeData.selected, $node);
         this._toggleSelectedClass($node, nodeData.selected);
+
+        nodeData.disabled && this.setAria('disabled', nodeData.disabled, $node);
```

Comment:
It's not a good practice to use logical operator instead of usual statement
https://github.com/airbnb/javascript#control-statements--value-selection
```suggestion
        if(nodeData.disabled) {
            this.setAria('disabled', nodeData.disabled, $node);
        }
```